dart/lib/src/transport/http_transport.dart

```diff
@@ -103,11 +103,12 @@ class _CredentialBuilder {
       : _authHeader = authHeader,
         _clock = clock;
 
-  factory _CredentialBuilder(Dsn dsn, String clientId, ClockProvider clock) {
+  factory _CredentialBuilder(
```

Comment:
renamed to align with the other methods that also takes a `sdkIdentifier` as param